examples/interactive/cli-platform/Effect.roc

```diff
@@ -1,10 +1,28 @@
 hosted Effect
-    exposes [Effect, after, map, always, forever, loop, putLine, getLine, sendRequest]
-    imports [InternalHttp.{ Request, Response }]
+    exposes [
+        Effect,
+        after,
+        map,
+        always,
+        forever,
+        loop,
+        stdoutLine,
+        stderrLine,
+        stdinLine,
+        sendRequest,
+        fileReadBytes,
+        fileWriteUtf8,
+        fileWriteBytes,
+    ]
+    imports [InternalHttp.{ Request, Response }, InternalFile]
     generates Effect with [after, map, always, forever, loop]
 
-putLine : Str -> Effect {}
+stdoutLine : Str -> Effect {}
+stderrLine : Str -> Effect {}
+stdinLine : Effect Str
 
-getLine : Effect Str
+fileWriteBytes : List U8, List U8 -> Effect (Result {} InternalFile.WriteErr)
```

Comment:
Why is there file-related stuff in Effect? Why no `fileReadUtf8`?

Reply:
`fileReadUtf8` would just be a `fileReadBytes` followed by validating that the bytes are UTF-8, and we can do that in Roc with `Str.fromUtf8`!

Reply:
Gotcha. Pardon my ignorance here, but why are these File-related types in this Effect interface at all?